api/api.yaml

```diff
@@ -139,6 +139,32 @@ paths:
         '500':
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
 
+  /control/ping:
+    post:
+      tags: ['Control']
+      operationId: send_ping
+      description: >-
+        Control operation to send ping
```

Comment:
API to send an IPv4 and/or IPv6 ICMP Echo Request(s).

Reply:
updated